tests/test_unclassified_job.py

```diff
@@ -26,3 +25,23 @@ def test_open_unclassified_failure_log(base_url, selenium):
     treeherder_page.open_next_unclassified_failure()
     logviewer_page = treeherder_page.job_details.open_logviewer()
     assert logviewer_page.is_job_status_visible
+
+
+@pytest.mark.nondestructive
+def test_view_unclassified_jobs(base_url, selenium):
+    page = TreeherderPage(selenium, base_url).open()
+    all_jobs = list(page.all_jobs)
+
+    page.filter_unclassified_jobs()
+    filtered_jobs = list(page.all_jobs)
+    jobstatus = page.result_sets[0].jobs[0].title
+
+    assert not all_jobs == filtered_jobs
```

Comment:
I think we should examine this to ensure it's doing what we're expecting. It might be that these lists would not be equivalent even if the jobs were not filtered. I say this because if the DOM is rewritten then the elements will be different but with identical content. Maybe try replacing the filter action with a page refresh to see if this assert fails. If it it doesn't then we may need to do a deeper comparison of the lists.